config/src/main/java/com/typesafe/config/ConfigMemorySize.java

```diff
@@ -0,0 +1,49 @@
+/**
+ *   Copyright (C) 2015 Typesafe Inc. <http://typesafe.com>
+ */
+package com.typesafe.config;
+
+/**
+ * An immutable class representing an amount of memory.  Use
+ * static factory methods such as {@link
+ * ConfigMemorySize#ofBytes(long)} to create instances.
+ */
+public final class ConfigMemorySize {
```

Comment:
Hmm, it isn't a lot of other stuff, it's ALL the other stuff I believe. I think this type is supposed to be config-specific; a general memory size type would belong in some more general library.